module.mk

```diff
@@ -106,10 +106,10 @@ ifneq (,$(filter "$(ARCH_NAME)","wormhole" "wormhole_b0"))
 endif
 
 set_up_kernels:
-	python3 $(TT_METAL_HOME)/scripts/set_up_kernels.py --short prepare
+	$(MAKE) -f $(TT_METAL_HOME)/tt_metal/hw/Makefile-runtime -C $(TT_METAL_HOME)/tt_metal/hw prepare
```

Comment:
I ideally wanted to make these scripts tested by consequence by injecting them into the build process here. But I guess this wouldn't always work

Checking functionality of these scripts should probably be something in CI/CD.

Reply:
Yes, it's part of setting things up for the eventual install-by-wheel flow for metal that we should (and if I was dictator, I would change that to "WILL have"). Buda has similar requirements, but to my current knowledge it does that by hiding it within some `__init__.py` files at the top of tree.

There should be uses for it in the `.github/workflows/eager-package-main.yaml`.

Right now, our leadership has stressed that functionality + performance via source install are of utmost importance. So, the wheel flow has taken a backseat. The purpose of this pipeline is to guarantee we don't regress.

Please let me know your thoughts.